src/Symfony/Component/String/Slugger/SluggerInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String\Slugger;
+
+use Symfony\Component\String\AbstractUnicodeString;
+
+/**
+ * Creates a URL-friendly slug from a given string.
+ *
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * @experimental in 5.0
+ */
+interface SluggerInterface
+{
+    /**
+     * Creates a slug based on a given string and an given locale, using appropriate
```

Comment:
`an given locale` -> `a given locale`  ... although I'd reword it a bit to avoid repeating "a given":

`Creates a slug for the given string and locale, using appropriate transliteration when needed.`